src/components/Table/Table.less

```diff
@@ -153,6 +156,10 @@
 				opacity: 1;
 			}
 
+			&.@{prefix}-Table-is-resizing > .@{prefix}-Table-Th-inner > .@{prefix}-Table-Th-inner-resize {
```

Comment:
stops the resizer from losing hover and growing/shrinking as it is dragged around